conans/client/cmd/export.py

```diff
@@ -76,10 +77,11 @@ def _capture_export_scm_data(conanfile, conanfile_dir, destination_folder, outpu
     captured_revision = scm_data.capture_revision
 
     scm = SCM(scm_data, conanfile_dir, output)
-    if scm_data.capture_origin or scm_data.capture_revision:
-        # Generate the scm_folder.txt file pointing to the src_path
-        src_path = scm.get_repo_root()
-        save(scm_src_file, src_path.replace("\\", "/"))
+    if get_bool_from_text(os.environ.get("USE_SCM_FOLER", "True")):
```

Comment:
These changes are here just to be able to run tests without the `scm_folder.txt` file.

Reply:
USE_SCM_FOLER => USE_SCM_FOLDER?

Reply:
If this is only for testing I don't like it, but we could consider this a feature naming in properly: `CONAN_USE_SCM_FOLDER`. To discuss.

Reply:
Right now it is only to make these tests possible. I want you all to review the tests, check the failing ones, they are showing some bugs that cannot be fixed without adding something else to the model.